scala-thoughts.md

```diff
@@ -0,0 +1,51 @@
+### What's easy to do in Scala? What's not?
+
+Although Scala users can choose between imperative and functional programming, I 
+find it much easier to use functional programming because the code is much more 
+concise. Moreover, Scala provides a lot of support for functional methods like 
+filter, partition, map, etc. 
```

Comment:
Definitely agree. It took some time to get back into the mindset of functional programming but things like pattern-matching make the code very concise and readable.